tests/runtests.pl

```diff
@@ -15,17 +15,20 @@
 my $output = `id`;
 $output =~ /uid=\d+\((\w+)\).*/;
 my $unix_user = $1;
-$output =~ /context=(\w+):(\w+):(\w+)/;
+$output =~ /context=(\w+):(\w+):(\w+):(\w+)-([\w:\.]+)/;
 my $selinux_user = $1;
 my $selinux_role = $2;
 my $selinux_type = $3;
+my $selinux_sen = $4;
+my $selinux_clr = $5;
 
 # Sanity checks prior to test execution.
 die ("These tests are intended to be run as root\n") unless $unix_user eq "root";
 die ("The auditctl tool cannot be found\n") unless which "auditctl";
 
 print "Running as   user    $unix_user\n";
 print "        with context $selinux_user:$selinux_role:$selinux_type\n";
+print "        with sensitivity and clearance $selinux_sen $selinux_clr\n";
```

Comment:
Let's print the SELinux context in the format that everyone expects, e.g. 'ps -Z', "$selinux_user:$selinux_role:$selinux_type:$selinux_eff:$selinux_clr".